runtime/runtime.go

```diff
@@ -781,6 +794,10 @@ func (r *interpreterRuntime) meteringInterpreterOptions(runtimeInterface Interfa
 			return
 		}
 
+		err := runtimeInterface.SetComputationUsed(used)
```

Comment:
The call to the runtime interface should be wrapped, just like in all other cases:

```suggestion
		var err error
		wrapPanic(func() {
			err = runtimeInterface.SetComputationUsed(used)
		})
```